src/frequenz/sdk/config/_util.py

```diff
@@ -0,0 +1,39 @@
+# License: MIT
+# Copyright © 2024 Frequenz Energy-as-a-Service GmbH
+
+"""Utilities to deal with configuration."""
+
+from collections.abc import Mapping
+from typing import Any, TypeVar, cast
+
+from marshmallow_dataclass import class_schema
+
+T = TypeVar("T")
+"""Type variable for configuration classes."""
+
+
+def load_config(
```

Comment:
Maybe load_marshmallow_config ?

Reply:
I'm not sure if it is worth adding all of that to the name of the function, after all the idea was to make the loading short and neat. Also, this won't be able to load using a plain `marshmallow.Schema`, it actually needs a `marshmallow_dataclass.dataclass`, so if we want to make it unambigous it would have to be at least `load_marshmallow_dataclass_config()` :grimacing:. At least for me if I read `load_marshmallow_config()` I would expect it to take a plain `marshmallow.Schema`.

Reply:
Updated!